src/core/prompts/system.ts

```diff
@@ -40,6 +40,17 @@ export function getPromptComponent(
 	return component
 }
 
+// Helper function to generate morph instructions when available
+function getMorphInstructions(cwd: string, supportsComputerUse: boolean, settings?: Record<string, any>): string {
```

Comment:
Please move this to a separate file (to reduce the likelihood of merge conflicts). That goes for all functions added.